types/meteor/index.d.ts

```diff
@@ -2145,3 +2145,67 @@ declare module "meteor/accounts-base" {
         function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
     }
 }
+
+declare module "meteor/webapp" {
```

Comment:
There are some style problems that seem pervasive to this module -- the first is that this should probably be in a `meteor/webapp.d.ts` file instead of in a `declare module`. These could wait until a later PR though.